Core/scripts/dirac-install.py

```diff
@@ -2623,7 +2631,9 @@ def createSymbolicLink():
     logNOTICE("Skipping installing DIRAC")
 
   # we install with DIRACOS from v7rX DIRAC release
-  if cliParams.diracOS or int(releaseConfig.prjRelCFG['DIRAC'].keys()[0][1]) > 6:
+  if cliParams.diracOS \
+     or isinstance(releaseConfig.prjRelCFG['DIRAC'].keys()[0][1], str_type) \
```

Comment:
Is `prjRelCFG['DIRAC']` is a dictionary? Then this doesn't work in python3, I think, because keys() returns a "view" and not a list.

Reply:
There is a separate PR for making dirac-install to work on python 3: https://github.com/DIRACGrid/DIRAC/pull/4411
@zmathe do you want to have a look?